### ISSUE TITLE
Feature request: Resend action for new subscribers

### ISSUE THREAD

**Issue by dahaupt** [in2code-de/luxletter#17]
A customer has requested a possibility to send a newsletter for new subscribers at a later date. Is this a generally useful enhancement for the extension?

I already created a fast proof-of-concept using some of the existing methods: ad3e5a0

**Comment by einpraegsam** [comment]
Sounds good to me. If you open a PR I will double-check the functionality.

**Comment by einpraegsam** [comment]
I have thought about this request and the more I dig into it the more I think it had to be a feature of the registration extension. 
While you add new newsletters, a queue is build by saving the newsletter with receivers (given at the time from a fe_group).
In future new frontendusers are registered and added to the group. But the newsletter with its queue is already generated. So the frontenduser registration should have a possibility to add new frontendusers to the queue. 
What do you think?

**Comment by einpraegsam** [comment]
So, now I added to new functions to luxletter that can be called from a third party extension to send an existing newsletter to a new registered user. I also described it here: https://github.com/in2code-de/luxletter/blob/develop/Documentation/Tech/Index.md#add-new-users-to-the-queue

Now it's possible to use a finisher in e.g. femanager that will send a defined or the latest newsletter to the new registered person. Nevertheless someone has to generated such a finisher next.
